Web API's/WebAPI/StudentProjectMVC/Controllers/StudentController.cs

```diff
@@ -11,19 +11,34 @@ namespace StudentProjectMVC.Controllers
 {
     public class StudentController : Controller
     {
-        StudentLogic studentRepository = new StudentLogic(new Context());
-        CourseLogic courseRepository = new CourseLogic(new Context());
+        StudentLogic studentRepository = new StudentLogic();
+        CourseLogic courseRepository = new CourseLogic();
 
         public ActionResult StudentList()
         {
             //Getting student list using studentContext object and passing it to GetStudentCourse method to get the list of courses of students.
             var viewModel = studentRepository.GetStudentCourse(studentRepository.Get()).ToList();
-            //Making an Object of StudentCourseViewModel and passing it directly to the view.
-            return View(new StudentCourseViewModel
-            {
-                StudentViewModels = viewModel,
-                CoursesList = courseRepository.Get().ToList()
-            });
+            return View();
+        }
+
+        [HttpPost]
+        public ActionResult AjaxGetJsonData()
```

Comment:
ajaxgetjsondata of what?Method name should describe the main functionality.It Should be like LoadStudent/GetStudentsData/GetStudentsJson